### ISSUE TITLE
Make Dynamic Variables( $(circus.wid) and freinds ) available everywhere

### ISSUE THREAD

**Issue by ayashjorden** [circus-tent/circus#960]
Hi,
My specific case is to have seperate log file (std*_stream.filename) per worker.
I've tried setting ```stdout_stream.filename = /tmp/log_$(circus.wid).log``` but no success (got a file named with Dollar sign and the brackets).

Are the dynamic variables available in another way?
Is there another way to create log file per worker?(not by passing an option in the cmd/args config)

Thank you,
Yarden

**Comment by k4nar** [comment]
Indeed, we should make this possible, I'm even surprised that it's not :) .
